client/src/Notes.js

```diff
@@ -0,0 +1,18 @@
+export default function Notes({ onChangeNotes, activeNotes }) {
+  return (
+    <div className="column-full">
+      <label className="margin-bottom-1 d-block" htmlFor="formNotes">
```

Comment:
You can wrap the label around the textarea and avoid using htmlFor or textarea.id.